src/Microsoft.Extensions.Logging.AzureWebApps/AzureWebAppsLoggerProvider.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.Logging.Abstractions.Internal;
+using Microsoft.Extensions.Logging.AzureWebApps.Internal;
+
+namespace Microsoft.Extensions.Logging.AzureWebApps
+{
+    /// <summary>
+    /// Logger provider for Azure WebApps
+    /// </summary>
+    public class AzureWebAppsLoggerProvider : ILoggerProvider
```

Comment:
Correct, the provider is not responsible for reloading the logger. Each internal logger (file, blob, etc) will be responsible for that.